### ISSUE TITLE
Touch doesn't stop inactivity alarm

### ISSUE THREAD

**Issue by mha1** [EdgeTX/edgetx#1767]
Not really worth to mention ...

Radiomaster TX16s, fw 2.0.7-rc1
Inactivity alarm set 15 minutes

**Comment by eshifri** [comment]
AFAIK inactivity alarm is looking at he controls (steaks and switches) only.

**Comment by mha1** [comment]
Yes, but with touch there is one additional means of  input. My use case: sitting in front of the radio and editing stuff. It starts beeping as if the radio was abandoned. Really no big deal as it is easy to push a stick. If it's easy to add touch reset the inactivity timer I'd do it.

**Comment by eshifri** [comment]
I'm not arguing with you suggestion to fix this. :-)
I was just saying that this is not new: the same happened in OTX with physical buttons.

**Comment by mha1** [comment]
This resets the inactivity alarm timer at any touch event. Works in a private build. Any concerns?

![Capture](https://user-images.githubusercontent.com/5615068/160905535-e0b9ca98-e07a-4fe2-b1d4-40868426f864.PNG)

**Comment by eshifri** [comment]
Looks good to me. Just submit a PR.
